packages/lit-dev-content/site/tutorials/content/working-with-lists/00.md

```diff
@@ -0,0 +1,33 @@
+This tutorial will be exploring different ways of rendering a list of
+items in Lit!
+
+## Previous knowledge
+It's recommended to have an understanding of
+[defining a component](/docs/components/defining/),
+[rendering](/docs/components/rendering/),and
+[reactive properties](/docs/components/properties/) before
+working on this tutorial.<br>
+Recommended prerequisite: [Intro to Lit](/tutorials/intro-to-lit/).
+
+
+## Rendering a list
+Begin by rendering a list of names from an array. The example already has set
+up a custom element with the `names` property that is an array of string.
+
+The `map` array prototype method can be utilized to generate a list item
+for each item in the array. Place the code snippet below in the return of the
+`render` method within the unordered list (`<ul></ul>`).
+
+{% switchable-sample %}
+
+```ts
+// my-element.ts
+${this.names.map((name) => html`<li>${name}</li>`)}
+```
+
+```js
+// my-element.js
+${this.names.map((name) => html`<li>${name}</li>`)}
```

Comment:
A little more description of what's happening here would be great. In particular, this is leveraging Lit's (super power?) ability easily to compose templates.

Reply:
Good call, especially since this one's geared toward beginners. Added a note below the code snippet.